src/frame_buffer.rs

```diff
@@ -165,28 +169,74 @@ impl<'a> FrameBuffer<'a> {
         self.handle
     }
 
-    // TODO: this should probably be part of Header.  It's only here
-    // right now to allow access to both struct's internals, but it won't
+    fn get_channel(&self, name: &str) -> Option<Channel> {
```

Comment:
Since this abuses `Channel`, it deserves a comment explaining that it needs to stay private unless it is rewritten to expose something closer an authentic OpenEXR `Slice`.